<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0"><widget class="QLabel" name="labelTheme"><property name="text"><string>Theme</string></property></widget></item>
   <item row="0" column="1"><widget class="QComboBox" name="comboBoxTheme"/></item>
   <item row="1" column="0"><widget class="QLabel" name="labelAutosave"><property name="text"><string>Autosave</string></property></widget></item>
   <item row="1" column="1"><widget class="QCheckBox" name="checkBoxAutosave"/></item>
   <item row="2" column="0" colspan="2"><widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons"><set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set></property>
   </widget></item>
  </layout>
 </widget>
</ui>
